language/stdlib/modules/LibraSystem.move

```diff
@@ -394,10 +395,16 @@ module LibraSystem {
         pragma opaque;
         aborts_if false;
         let new_validator_config = ValidatorConfig::spec_get_config(validators[i].addr);
+        // Amazingly, it is able to prove this because get_validator_index_ ensures it
```

Comment:
Use `///` (here and elsewhere) and don't use expression like "amazingly"? This is documentation part of the artifact, not a note paper.